src/lib/apiTypesV2/Attribute.cpp

```diff
@@ -125,6 +125,7 @@ void Attribute::fill(const QueryContextResponse& qcrs, const std::string& attrNa
     contextAttributeP = NULL;
     // Look for the attribute by name
 
+    std::string NotExistingAtrr;
```

Comment:
```suggestion
    std::string notExistingAtrr;
```

as variables use camelCase

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4097/commits/2dbb2ed15aee6867d16807ce665a18c7bf282c50